### DISCUSSION TITLE
Background colors don’t print

### DISCUSSION THREAD

**Discussion by raystek** [kimai/kimai#2296] [Questions & Answers]
I customized the html invoice template by adding background colors to the table cells. When I print the invoice or export the html  to pdf, the colors don’t show up. This might be a CSS question. Any advice would be greatly appreciated.

**Comment by kevinpapst** [comment]
1. How is this related to Kimai?
2. How should someone help without example Code of what you already tried? Please update your question and show the relevant twig/html code.
3. https://mpdf.github.io/what-else-can-i-do/backgrounds-borders.html

**Reply by raystek** [reply]
I use the html template, not PDF. For example  I added background color to some of table cells: <td style="background-color: dodgerblue;">blah blah blah</td>

The generated html invoice shows the correct color but the colors don't appear on printed output. I have done some research and it seems to be related to bootstrap and/or the @media print {} CSS settings.

**Reply by kevinpapst** [reply]
> or export the html to pdf
> I use the html template, not PDF.

Please be precise when asking questions (these two points sound like the complete opposite) , I spent my free time trying to help, which is only possible with as many information as possible.

So I assume now that you are talking about your browsers print function. In that case you are right with your last point and I was right with 1 😁 It has really nothing to do with Kimai at all, but is only a CSS issue.

I know the pain: creating HTML templates for printing can be annoying. Safari has a developer feature to toggle the media class for printing, so you can easily debug your print stylesheets. I guess Firefox and Chrome should have something like that as well. Maybe that helps...
 <img width="169" alt="Bildschirmfoto 2021-01-27 um 22 23 32" src="https://user-images.githubusercontent.com/533162/106055890-4b7e8d80-60ee-11eb-9fe7-c33318971a18.png">
